aws/resource_aws_ecs_task_definition.go

```diff
@@ -396,6 +455,10 @@ func resourceAwsEcsTaskDefinitionRead(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	if err := d.Set("proxy_configuration", flattenProxyConfiguration(taskDefinition.ProxyConfiguration)); err != nil {
+		return err
```

Comment:
It appears the rest of the resource may not be updated for this yet, but preferably we should return some error messaging context for operators and code maintainers when something goes wrong so the issue can be more easily triaged. e.g.

```suggestion
		return fmt.Errorf("error setting proxy_configuration: %s", err)
```